src/components/slider/Slider.module.css

```
@@ -0,0 +1,261 @@
+/* Slider.module.css - Uses PeakFit Theme System */
+
+.container {
+  width: 100%;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
```

Biome (2.4.4)

[error] Unknown property is not allowed.

(lint/correctness/noUnknownProperty)